settings.py

```diff
@@ -48,7 +48,6 @@
 MIDDLEWARE = [
     'django.middleware.security.SecurityMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
```

Comment:
The `csrf` middleware is loaded when seqr is in the non-debugging mode.

Reply:
CSRF middleware is crucial for app security and so always should be loaded, that was a PR you approved last week